src/aof.c

```diff
@@ -1552,7 +1552,7 @@ int rewriteAppendOnlyFile(char *filename) {
 
     if (server.aof_use_rdb_preamble) {
         int error;
-        if (rdbSaveRio(&aof,&error,RDBFLAGS_AOF_PREAMBLE,NULL) == C_ERR) {
+        if (rdbSaveRio(SLAVE_REQ_NONE, &aof,&error,RDBFLAGS_AOF_PREAMBLE,NULL) == C_ERR) {
```

Comment:
This space feels odd, and there are two more below.
```suggestion
        if (rdbSaveRio(SLAVE_REQ_NONE,&aof,&error,RDBFLAGS_AOF_PREAMBLE,NULL) == C_ERR) {
```